### PR TITLE
chore: Update zora contract

### PR DESCRIPTION
**Pull request by abcrane123**
**What changed? Why?**
- update zora contract

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: src/components/TransactionWrapper.tsx [hunk 1]
```diff
@@ -16,27 +27,43 @@ type TransactionWrapperParams = {
...
-      args: [],
-    },
-  ] as ContractFunctionParameters[];
+  ] as unknown as ContractFunctionParameters[];
+
+  const handleError = (err: TransactionError) => {
+    console.error('Transaction error:', err);
+  };
+
+  const handleSuccess = (response: any) => {
```
Comment (abcrane123) [inline src/components/TransactionWrapper.tsx:54]: we need to export `TransactionResponse` from onchainkit

Comment (abcrane123) [reply src/components/TransactionWrapper.tsx:54]: https://github.com/coinbase/onchainkit/pull/1007

File: src/components/TransactionWrapper.tsx [hunk 2]
```diff
@@ -6,8 +6,18 @@ import {
   TransactionStatusAction,
   TransactionStatusLabel,
 } from '@coinbase/onchainkit/transaction';
-import { clickContractAbi, clickContractAddress } from 'src/constants';
+import type { TransactionError } from '@coinbase/onchainkit/transaction';
+import { mintABI, mintContractAddress } from 'src/constants';
 import type { Address, ContractFunctionParameters } from 'viem';
+import { parseEther } from 'viem';
+
+const BASE_SEPOLIA_CHAIN_ID = 84532;
```
Comment (Zizzamia) [inline src/components/TransactionWrapper.tsx:14]: let's move this to constant file

File: src/components/TransactionWrapper.tsx [hunk 3]
```diff
@@ -6,8 +6,18 @@ import {
...
+
+const collectionAddress = '0xd6915560d3bb24aec04dc42ef409921ed1931510';
+const tokenId = '1';
+const quantity = '1';
+const mintReferral = '0x0000000000000000000000000000000000000000';
+const comment = 'testing';
```
Comment (Zizzamia) [inline src/components/TransactionWrapper.tsx:20]: Same here, let's use more polish constant.

**Review by Zizzamia — APPROVED** [review]

**Pull request merged**